open_academy/models/session.py

```diff
@@ -46,3 +49,12 @@ def _check_attendees_ids(self):
         for record in self:
             if record.instructor_id in record.attendees_ids:
                 raise ValidationError("An instructor cannot be an attendee of his own session")
+
+    @api.onchange('duration', 'start_date')
+    def _onchange_end_date(self):
+        difference = timedelta(days=self.duration)
```

Comment:
Add the L55 - 56 to a for block something like 
```suggestion python
        for record in self:
            difference = timedelta(days=record.duration)```